### ISSUE TITLE
Upload Errors with FC24-19

### ISSUE THREAD

**Issue by jessicagood** [StraboSpot/StraboSpot-Mobile#197]
Test uploading using FC-19. Status messages duplicated and in wrong order. Download project first then Ok to delete off server.
![upload-status-messages](https://user-images.githubusercontent.com/1493985/98737401-5f261d80-2363-11eb-8f77-03e5d6470ef5.JPG)

**Comment by sdamaraju** [comment]
![Screen Shot 2020-11-26 at 9 39 39 AM](https://user-images.githubusercontent.com/28953643/100307354-9df6dd00-2fcb-11eb-8f49-032198cc514c.png)

@jessicagood @panamint55 This is how I have the status box messages, when I tried to upload the project.

Only Change I had to do, was comment out the resizing logic, as it was continuously giving me the error .. (It could be a different issue, unrelated to the messages though.)

<img width="455" alt="Screen Shot 2020-11-26 at 8 30 16 AM" src="https://user-images.githubusercontent.com/28953643/100307416-c8e13100-2fcb-11eb-9b85-a7eae1ef2013.png">

I'm deleting the project from strabo.org again, because, the images uploaded are not resized. But the status messages look fine on my system

**Comment by jessicagood** [comment]
@sdamaraju Ok, thanks for running that test. I got the same error with the image resizer and rolled back to v.1.3.0. People said they weren't getting that error with iOS so I hadn't pushed up the rollback change but sounds like I should. @panamint55 Since it is working well for me and @sdamaraju for this project we'll have to figure what is different with your setup that the upload isn't working as well.

**Comment by sdamaraju** [comment]
Hi Jessica,

I tried to roll back to 1.2 version and still it did not work for me.
I removed the node modules and did npm install too.

Thanks,
Sai

**Comment by sdamaraju** [comment]
@jessicagood, my bad, I think I was wrong, when I said, I used my device, I think I actually used the emulator, initially.

Now that I tried to upload the project using the device, I see that status section has grown really long with the details and it's not scrollable. Hence I do not see the OK button in the end. I guess making status box scrollable should solve that issue. Other than that, I do not see any messages getting messed up or duplicated.

I will look into this and try to make the status box scrollable.
![image](https://user-images.githubusercontent.com/28953643/100919977-886c4080-3500-11eb-9d49-d33050e9d0ee.png)

Thanks,
Sai
